examples/travis.example.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 4
+  - 5
```

Comment:
Yeah, on the other hand this means we are running tests twice on the same platform, so I was thinking we should rather update this manually when a new version of node is released, so we exactly control what versions we are running on.